include/mgp.hpp

```diff
@@ -2390,6 +2322,27 @@ inline void Node::AddLabel(const std::string_view label) {
   mgp::vertex_add_label(this->ptr_, mgp_label{.name = label.data()});
 }
 
+inline std::map<std::string, Value> Node::Properties() const {
+  mgp_properties_iterator *properties_iterator = mgp::vertex_iter_properties(ptr_, memory);
+  std::map<std::string, Value> property_map_;
+  for (auto *property = mgp::properties_iterator_get(properties_iterator); property;
+       property = mgp::properties_iterator_next(properties_iterator)) {
+    auto value = Value(property->value);
+    property_map_.emplace(std::string(property->name), value);
+  }
+  mgp::properties_iterator_destroy(properties_iterator);
+  return property_map_;
+}
+
+inline void Node::SetProperty(std::string property, Value value) const {
+  mgp::vertex_set_property(ptr_, property.data(), value.ptr());
+}
+
+inline Value Node::GetProperty(const std::string &property) const {
+  mgp_value *vertex_prop = mgp::vertex_get_property(ptr_, property.data(), memory);
+  return Value(&vertex_prop);
```

Comment:
Why do you need to get the address of a pointer? Isn't `Value(vertex_prop)` would suffice?

Reply:
Can you check inline Value::Value(mgp_value **ptr) : ptr_(*ptr) {} for more extensive comment? Also check Relationship::GetProperty. If that doesn't make sense, I will change it.